scripts/lint-md-links.ts

```diff
@@ -2,23 +2,26 @@
 import { execSync } from 'child_process';
 import { globSync } from 'glob';
 
+const { log } = console;
+
 (() => {
   const mdFiles = globSync('**/*.md', {
     ignore: [
       '**/node_modules/**',
-      'apps/docs/src/api/**', // generated api
-      '**/CHANGELOG.md',
-      'apps/demo-nextjs/**',
+      'apps/demo/**',
```

Comment:
```suggestion
      'apps/demo-**',
```

`apps/demo/**` doesn't match any projects, did you want to ignore all demos?

Reply:
@petertonysmith94 I had to make it like this to work: https://github.com/FuelLabs/fuels-ts/pull/3430/commits/a7acbec0b5fe8c6bc28487826a6c32574fe63929